content/en/docs/tasks/tls/managing-tls-in-a-cluster.md

```diff
@@ -37,13 +37,25 @@ these certificates will validate against the cluster root CA.
 
 ## Trusting TLS in a Cluster
 
-Trusting the custom CA from an application running as a pod usually requires
+Trusting the [custom CA](#a-note-to-cluster-administrators) from an application running as a pod usually requires
 some extra application configuration. You will need to add the CA certificate
 bundle to the list of CA certificates that the TLS client or server trusts. For
 example, you would do this with a golang TLS config by parsing the certificate
 chain and adding the parsed certificates to the `RootCAs` field in the
 [`tls.Config`](https://godoc.org/crypto/tls#Config) struct.
 
+{{< note >}}
+Even though the custom CA certificate may be included in the filesystem (in the
+ConfigMap `kube-root-ca.crt`),
+you should not use that certificate authority for any purpose other than to verify internal
+Kubernetes endpoints. An example of an internal Kubernetes endpoint is the
+Service named `kubernetes` in the default namespace.
+
+If you want to use a custom certificate authority for your workloads, you should generate
+that CA separately, and distribute its CA certificate using a ConfigMap that your pods have access
+to read.
+{{< /note >}}
+
 You can distribute the CA certificate as a
 [ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap) that your
 pods have access to use.
```

Comment:
The second paragraph of the note seems similar enough to the paragraph following it that I'm worried readers might end up confused (wondering, perhaps, what they have misunderstood).

Can we clarify? I'd be tempted to put the note at the end of the section, before “Requesting a certificate”, and to use  `caution` callout rather than `note`. _(Why `caution`? Because it is letting readers know about a detail relevant to good information security practises)._

Reply:
Maybe we should remove `You can distribute the CA certificate as a [ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap) that your pods have access to use.` all together and add the link to the note.

Reply:
Sounds good to me.